js/script.js

```diff
@@ -1,205 +0,0 @@
-const startForm = {
-  // UI DECLARATION
-  ui: {
-    form: document.querySelector(".form"),
-    fields: document.querySelectorAll(".input-field"),
-    inputs: {
-      name: document.querySelector(".name"),
-      email: document.querySelector(".email"),
-      message: document.querySelector(".message")
-    },
-    button: document.querySelector(".button"),
-  },
-
-  // FUNCTIONS
-  functions: {
-    formValidation: () => {
-      const name = startForm.ui.inputs.name;
-      const email = startForm.ui.inputs.email;
-      const message = startForm.ui.inputs.message;
-      const input = startForm.ui.fields;
-      const button = startForm.ui.button;
-      let erros = 0;
```

Comment:
If you are not reassigning this variable it should be declared as a const, and the name should be "errors" instead of "erros"